androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/common/Overlay.kt

```diff
@@ -0,0 +1,18 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.screen.common
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+
+@Composable
+fun Overlay(
+    modifier: Modifier,
+    colors: List<Color>
+) {
```

Comment:
Same here. The modifier should be an optional parameter:

```suggestion
fun Overlay(
    colors: List<Color>,
	modifier: Modifier = Modifier,
) {
```

Reply:
Fixed in acd211abce503054711d0c24171bd83cc8117768